src/UIState/NumberCollectorState.cpp

```diff
@@ -10,26 +10,21 @@ void NumCollectorState::handleKey(char key) {
     numDigits++;
     handleDigit(key - '0');
     if (collectedEnoughDigits()) {
```

Comment:
I thought we discussed removing `collectedEnoughDigits()`. Will that happen later?

Reply:
Yes, it is on its way, probably tonight!